doc/netplan-yaml.md

```diff
@@ -374,11 +374,19 @@ Match devices by MAC when setting options like: `wakeonlan` or `*-offload`.
   > (networkd backend only) Allow the specified interface to be configured even
   > if it has no carrier.
 
-- **critical** (bool)
+- **critical** (scalar)
```

Comment:
We'd need to keep this as "(bool or scalar)", making use of `handle_generic_bool()` internally in `parse.c`, checking for the error condition in the `GError** error` parameter. Then falling back to "scalar" (and validating the input to be one of the accepted strings) if a bool cannot be parsed.